src/controllers/metadata.ts

```diff
@@ -0,0 +1,55 @@
+import { Context } from 'koa'
+import { Controller, Ctx, Get, Params } from 'amala'
+import { ethers } from 'ethers'
+import { goerliProvider } from '@/helpers/providers'
+import Metadata from '@/validators/Metadata'
+import abiForName from '@/helpers/contracts/abiForName'
+import data from '@/data'
+import env from '@/helpers/env'
+import getBadge from '@/helpers/getBadge'
+import getContract from '@/helpers/getContract'
+import getOriginalContractName from '@/helpers/getOriginalContractName'
+import nodeHtmlToImage from 'node-html-to-image'
+import renderReact from '@/helpers/renderReact'
+
+@Controller('/')
+export default class LoginController {
+  @Get('/image/:tokenAddress/:tokenId')
+  async image(@Ctx() ctx: Context, @Params() params: Metadata) {
+    const { tokenAddress, tokenId } = params
+
+    const contract = new ethers.Contract(
```

Comment:
Can we import `Contract` without `ethers`?

Reply:
ty